### ISSUE TITLE
Create directories in the fuzzy search

### ISSUE THREAD

**Issue by TeoDev1611** [liuchengxu/vim-clap#668]
**Is your feature request related to a problem? Please describe.**
How create directorys in clap

**Describe the solution you'd like**
Create a command for add a directory or when i am writing create the directory style the file for example an_example_dir/

**Describe alternatives you've considered**
Or create a simple command or mapping if exist write in the documentation

**Comment by liuchengxu** [comment]
Which provider are you referring to? Basically, you can add the action support in that provider https://github.com/liuchengxu/vim-clap/issues/662#issuecomment-810836289 or enhance the current behavior.

**Comment by romgrk** [comment]
Filer I think? It's true that it would be convenient to be able to create a directory like this:
![Screenshot from 2021-04-07 22-07-09](https://user-images.githubusercontent.com/1423607/113957674-ab2f9e00-97ed-11eb-9746-47009bbd61ef.png)

**Comment by liuchengxu** [comment]
It's technically not hard for filer, but needs to come up with a complete and decent UX. A simple `edit` is enough for creating a file, but not sure it's also good for creating a directory.

**Comment by romgrk** [comment]
I would think something like create the directory & switch to it (staying in filer) would be interesting.

**Comment by TeoDev1611** [comment]
> Filer I think? It's true that it would be convenient to be able to create a directory like this:
> ![Screenshot from 2021-04-07 22-07-09](https://user-images.githubusercontent.com/1423607/113957674-ab2f9e00-97ed-11eb-9746-47009bbd61ef.png)

I try this but this not work 
![image](https://user-images.githubusercontent.com/73144037/114031727-430aa780-9841-11eb-8885-b68ba6741444.png)

**Comment by TeoDev1611** [comment]
Yes this is the idea, I want make vim-clap my first file explorer

**Comment by liuchengxu** [comment]
Not sure I understand it correctly, you mean switching to the newly created directory? but it's empty, you can only create a new file under that directory.

**Comment by TeoDev1611** [comment]
Apologies if you do not understand me English is not my main language.
The idea is to create directories like some file explorers do for example ``` example_dir/ ```and leave it in the same directory and then open the directory. Personally I think that entering the created directory is not necessary

**Comment by liuchengxu** [comment]
No worries, I'm not a native English speaker either :(.

You have to understand that filer is kind of limited at the moment compared to the traditional file explorer like nerdtree, for it can only show the top level of a directory and can't show all the items recursively in tree view. `example_dir/` can just be created accordingly, but I'm unsure what to do next.

**Comment by TeoDev1611** [comment]
The idea after create the dir can rename the dir and the files move etc.
